mysite/settings.py

```diff
@@ -159,7 +159,7 @@ def logfile_handler(
             # only log to the file, let some reverse-proxy handle console logs
             #
             # p. s. gunicorn already has own formatting, let it keep it
-            "handlers": ["logfile_minfmt"],
+            "handlers": ["console_minfmt", "logfile_minfmt"],
```

Comment:
handle this as well, before merge, ideally we wouldn't have gunicorn logs locally, only in render.com

Reply:
and why the heck they are different, like, what?

Reply:
is it a different python version? that would be a strange difference tbh

Reply:
or maybe render.com specifies some env variable or something

Reply:
ok, render.com somehow enables access-log by default, which is idk, wowzers